lib/coqProject_file.ml

```diff
@@ -225,6 +225,7 @@ let process_cmd_line ~warning_fn orig_dir parse_extra proj args =
     error "Use \"-arg -impredicative-set\" instead of \"-impredicative-set\""
 
   | "-Q" :: d :: lp :: r ->
+    let lp = if String.equal lp "Coq" then "Stdlib" else lp in
```

Comment:
Why do this for `-Q` but not `-R`?

Reply:
I guess because it's enough in practice for stdlib, but you're right, let's be consistent.